Runtime/RestrictiveEnvironment/DefaultStepLockHandling.cs

```diff
@@ -11,13 +11,34 @@ namespace Innoactive.Creator.Core.RestrictiveEnvironment
     /// </summary>
     public class DefaultStepLockHandling : StepLockHandlingStrategy
     {
+        /// <summary>
+        /// If you want to disable lock handling set this parameter to false.
+        /// </summary>
+        public const string EnableLockHandlingParameterName = "EnableLockHandling";
+
+        /// <summary>
+        /// To stop locking every LockableProperty at the beginning of the training set this parameter to false.
+        /// </summary>
+        public const string LockCourseOnStartParameterName = "LockCourseOnStart";
```

Comment:
Here it's called LockCourse**On**Start and in line 29 it's called **lockOnCourseStart**
And the other parameter is called lock**On**Course finished. It's a little confusing.
In #118 it is now written down as LockOnCourseStart as well, so I would suggest using that naming then.